docs/03-architecture/supported-browsers.mdx

```diff
@@ -5,25 +5,18 @@ description: Browser support and which JavaScript features are supported by Next
 
 Next.js supports **modern browsers** with zero configuration.
 
-- Chrome 64+
-- Edge 79+
-- Firefox 67+
-- Opera 51+
-- Safari 12+
+- Chrome 111+
+- Edge 111+
+- Firefox 111+
+- Safari 16.4+
 
 ## Browserslist
 
 If you would like to target specific browsers or features, Next.js supports [Browserslist](https://browsersl.ist) configuration in your `package.json` file. Next.js uses the following Browserslist configuration by default:
 
 ```json filename="package.json"
 {
-  "browserslist": [
-    "chrome 64",
-    "edge 79",
-    "firefox 67",
-    "opera 51",
-    "safari 12"
-  ]
+  "browserslist": ["chrome 111", "edge 111", "firefox 111", "safari 16.4"]
```

Comment:
also non blocking:  what are the oldest browsers that we actually support.  is there a minimium?

like would we support chrome 50?